mkdocs-nav.yml

```diff
@@ -165,6 +165,7 @@ nav:
           - "widgets/tabbed_content.md"
           - "widgets/tabs.md"
           - "widgets/text_area.md"
+          - "widgets/toast.md"
```

Comment:
My memory is hazy, but I'm sure we left toast out of the widget list as it's not really a "public-facing" widget; it's more of an internal support widget for the default implementation of `App.notify`.